shapeshift/web
# fix: revert feat: default fiat currency (#4813)

## Description



reverts https://github.com/shapeshift/web/pull/4813

this doesn't work as intended, and we aren't prepared to spend a significant amount of time debugging long tail edge cases

e.g. australian users with an `en_AU` system language were shown GBP. this is particularly offensive for those with convict blood.

## Pull Request Type


## Issue (if applicable)



n/a

## Risk

low - reverting a recently introduced change



## Testing

same as original PR

* clear localforage database
* refresh page with browser set to non-US locale
* see that USD freedom bucks is default currency



### Engineering

as above



### Operations

as above



## Screenshots (if applicable)
